### PR TITLE
INTGR-144

### PR DESCRIPTION
**Pull request by wajihqazi94**

### CODE REVIEW

File: src/pages/myGeotab/apiReference/method.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// import { RouteComponentProps } from 'react-router-dom';
+
+import { Header } from "@geotab/react-component-library";
+import { useParams } from "react-router-dom";
+
+export default function Method() {
+    const { methodId } = useParams();
+
+    console.log(sessionStorage);
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/method.tsx:9]: remove console logs or do you need them here?

File: src/pages/myGeotab/apiReference/methods.tsx [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+});
+
+export default function Methods() {
+    return (
+        <Page section={HeaderSections.MyGeotab} pageTitle={pageTitle} tableOfContents={pageSections}>
+            {methodItems}
+            {/* {pageSections.map((section) => <Accordion summary={section.summary} p={section.details} id={section.elementId}></Accordion>)} */}
+        </Page>
+        // <div>
+        //     <Header title="Methods"></Header>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methods.tsx:57]: remove commented out code

File: src/pages/myGeotab/apiReference/objects.tsx
Comment (kmets89) [inline src/pages/myGeotab/apiReference/objects.tsx:1]: remove commented out code, unused references, and console logs

File: src/pages/myGeotab/apiReference/methods.tsx
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methods.tsx:1]: removed unused imports

File: src/pages/myGeotab/apiReference/methodDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methodDetails.tsx:1]: unused

File: src/pages/myGeotab/apiReference/objects.tsx [hunk 4]
```diff
@@ -1,7 +1,58 @@
...
+
 export default function Objects() {
     return (
-        <div>
-            <h1>All classes go here I guess</h1>
-        </div>
+        <Page section={HeaderSections.MyGeotab} pageTitle={pageTitle} tableOfContents={pageSections}>
+            {objectItems}
+        </Page>
+        // <div>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/objects.tsx:53]: remove

File: src/pages/myGeotab/apiReference/renderStringWithUrl.tsx
Comment (kmets89) [inline src/pages/myGeotab/apiReference/renderStringWithUrl.tsx:1]: remove commented out code, unused import

File: src/pages/myGeotab/apiReference/methods.tsx [hunk 5]
```diff
@@ -1,7 +1,61 @@
...
+    };
+
+    pageSections.push(pageSectionObject);
+
     return (
-        <div>
-            <h1>Just so many methods</h1>
+        <div id={methodDetails[0]}>
+            <Header title={methodDetails[0]}>
+                <Link to={`/method/${methodDetails[0]}`}>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methods.tsx:41]: add  text-decoration: none; to these links so there's no underline on the button

File: src/pages/myGeotab/apiReference/methods.tsx [hunk 6]
```diff
@@ -1,7 +1,61 @@
...
+        "summary": methodDetails[0],
+        "details": RenderStringWithUrl(methodDetails[1].description)
+    };
+
+    pageSections.push(pageSectionObject);
+
     return (
-        <div>
-            <h1>Just so many methods</h1>
+        <div id={methodDetails[0]}>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methods.tsx:39]:
```suggestion
        <div  className="paragraph" id={methodDetails[0]}>
```
Need to add paragraph styling to details sections

File: src/pages/myGeotab/apiReference/objects.tsx [hunk 7]
```diff
@@ -1,7 +1,58 @@
...
+    let pageSectionObject = {
+        "elementId": objectDetails[0],
+        "summary": objectDetails[0],
+        "details": RenderStringWithUrl(objectDetails[1].description)
+    };
+
+    pageSections.push(pageSectionObject);
+
+    return ( 
+        <div id={objectDetails[0]}>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/objects.tsx:38]:
```suggestion
        <div  className="paragraph" id={objectDetails[0]}>
```

File: src/pages/myGeotab/apiReference/objects.tsx [hunk 8]
```diff
@@ -1,7 +1,58 @@
...
+            <p>{RenderStringWithUrl(objectDetails[1].description)}</p>
+        </div>
+    )
+})
+
+
 export default function Objects() {
     return (
-        <div>
-            <h1>All classes go here I guess</h1>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/objects.tsx:4]: according to figma should be h3

File: src/pages/myGeotab/apiReference/methods.tsx [hunk 9]
```diff
@@ -1,7 +1,61 @@
...
+            <Header title={methodDetails[0]}>
+                <Link to={`/method/${methodDetails[0]}`}>
+                    <Button>View</Button>
+                </Link>
+            </Header>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methods.tsx:44]: this should just be an h3

File: src/pages/myGeotab/apiReference/methodDetails.tsx
Comment (kmets89) [inline src/pages/myGeotab/apiReference/methodDetails.tsx:1]: is this being used?

File: src/pages/myGeotab/apiReference/method.tsx
Comment (kmets89) [inline src/pages/myGeotab/apiReference/method.tsx:1]: is this being used?

File: src/pages/myGeotab/apiReference/objects.tsx [hunk 10]
```diff
@@ -1,7 +1,58 @@
...
+        "elementId": objectDetails[0],
+        "summary": objectDetails[0],
+        "details": RenderStringWithUrl(objectDetails[1].description)
+    };
+
+    pageSections.push(pageSectionObject);
+
+    return ( 
+        <div id={objectDetails[0]}>
+            <Header title={objectDetails[0]}>
```
Comment (kmets89) [inline src/pages/myGeotab/apiReference/objects.tsx:39]: should just be an h3, also title needs to be left justified

**Review by kmets89 — CHANGES_REQUESTED** [review]

File: src/pages/myGeotab/apiReference/methodDetails.tsx
Comment (wajihqazi94) [reply src/pages/myGeotab/apiReference/methodDetails.tsx:1]: Will delete this. Not being used.

File: src/pages/myGeotab/apiReference/method.tsx
Comment (wajihqazi94) [reply src/pages/myGeotab/apiReference/method.tsx:1]: Yeah but this was worked on in INTGR-215.

**Pull request merged**